contracts/SL2RD.sol

```diff
@@ -132,8 +213,11 @@ contract SL2RD is
         );
     }
 
-    /// @notice Overrides ERC-721 safeTransferFrom function and ensures
-    /// only SHARE approved wallets can recieve the SL2RD slot NFT.
+    /// @notice Allows for an ERC-721 token to be transferred to a new address.
+    /// @dev Overrides the ERC721 version to add additional checks:
+    /// - Ensures recipient address is a SHARE approved wallet hash.
+    /// - Only authorizes the owner transfer of withheld slots, until all of the
+    /// community allocation has been distributed.
     function safeTransferFrom(
```

Comment:
no, because the super.transferFrom will call this inside of it. It would be redundant. Plus i am not entirely sure if that would prevent someone from trading their slot on their own, or not.